Uno.Gallery/Uno.Gallery.Shared/Extensions/ControlExtensions.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using Windows.UI.Xaml;
+
+namespace Uno.Gallery
+{
+	/// <summary>
+	/// Helper class for Controls Extensions.
+	/// </summary>
+	public static class ControlExtensions
+	{
+		public static readonly DependencyProperty ResourcePathProperty =
+		DependencyProperty.RegisterAttached(
+			"ResourcePath",
+			typeof(string),
+			typeof(ControlExtensions),
+			new PropertyMetadata(null, OnResourcePathChanged));
+
+		public static void SetResourcePath(FrameworkElement element, string value)
```

Comment:
Should it be Control instead of FrameWorkElement? Or just leaving it as it is is better?

Reply:
leave it targetting FrameworkElement, thats where the .Resources property is from

Reply:
you can also rename it to ResourceExtensions::OverridePath